server/helpers/db.js

```diff
@@ -102,12 +102,27 @@ const deleteOrganisation = async organisation_id => {
 };
 
 const getLessonsById = course_id => {
```

Comment:
Can you rename the function as the current name is confusing. Here you are loading lessons by course id and not by lesson id. So a better name would be getLessonsByCourseId.

Reply:
ok